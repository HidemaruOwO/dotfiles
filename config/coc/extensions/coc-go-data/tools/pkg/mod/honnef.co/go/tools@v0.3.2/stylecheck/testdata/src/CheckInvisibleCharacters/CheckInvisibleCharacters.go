// Package pkg ...
package pkg

var (
	a = ""  //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬" //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)
