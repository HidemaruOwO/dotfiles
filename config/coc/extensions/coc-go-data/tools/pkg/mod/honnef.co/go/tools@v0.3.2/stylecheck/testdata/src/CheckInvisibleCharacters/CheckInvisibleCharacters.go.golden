-- replace control character U+0007 with '\a' --
// Package pkg ...
package pkg

var (
	a = "\a" //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬\a" //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)

-- delete control character U+0007 --
// Package pkg ...
package pkg

var (
	a = ""   //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"   //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)

-- delete format character U+200B --
// Package pkg ...
package pkg

var (
	a = ""  //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `ZeroWidth` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"  //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)

-- replace format character U+200B with '\u200b' --
// Package pkg ...
package pkg

var (
	a = ""  //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero\u200bWidth` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"  //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)

-- delete all control characters --
// Package pkg ...
package pkg

var (
	a = "" //@ diag(`Unicode control character U+0007`)
	b = ""  //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"  //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)

-- replace all control characters with escape sequences --
// Package pkg ...
package pkg

var (
	a = ""      //@ diag(`Unicode control character U+0007`)
	b = "\a\x1a" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"  //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬​" //@ diag(`Unicode format and control characters`)
)


-- delete all format and control characters --
// Package pkg ...
package pkg

var (
	a = ""  //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬" //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬"  //@ diag(`Unicode format and control characters`)
)

-- replace all format and control characters with escape sequences --
// Package pkg ...
package pkg

var (
	a = ""  //@ diag(`Unicode control character U+0007`)
	b = "" //@ diag(`Unicode control characters`)
	c = "Test	test"
	d = `T
est`
	e = `Zero​Width` //@ diag(`Unicode format character U+200B`)
	f = "\u200b"
	g = "👩🏽‍🔬"        //@ diag(`Unicode control character U+0007`)
	h = "👩🏽‍🔬\a\u200b" //@ diag(`Unicode format and control characters`)
)
